sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -171,19 +174,35 @@ follows:
 
 Should you wish to learn more about creating new resources, you can visit
 the [Secrets YAML reference](https://docs.semaphoreci.com/article/51-secrets-yaml-reference),
-the [Dashboard YAML reference]()
+the [Dashboard YAML reference](https://docs.semaphoreci.com/article/60-dashboards-yaml-reference),
+the [Jobs YAML reference]()
```

Comment:
empty link

Reply:
@markoa The same as before: https://github.com/semaphoreci/docs/pull/49
I think that Darko and I will take care of it tomorrow.